CODE_OF_CONDUCT.md

```diff
@@ -23,23 +14,29 @@ All recipients of reports commit to:
     * **Node-RED:** <team@nodered.org>
     * **Webdriver.io:** [contact TSC reps](https://github.com/webdriverio/webdriverio/blob/HEAD/AUTHORS.md)
     * **Webhint:** <support@webhint.io>
-2. For every other OpenJS Foundation project, please email <report@lists.openjsf.org>. The Cross Project Council (CPC) is responsible for managing these reports.
+    
+    In case of a conflict of interest or if the report is not handled appropriately, you may [escalate the report][escalation] to the [Code of Conduct Team (CoC Team)][CoC Team] by emailing <coc-escalation@lists.openjsf.org> or by contacting any of its members.
+2.  **To report any other incident** (for example: in an OpenJS Foundation project not listed above, in a Collab Space, in the Cross Project Council, or during an event), please email the [Code of Conduct Team (CoC Team)][CoC Team] at <report@lists.openjsf.org> or contact any of its members.
+
+### Appeal a decision
 
+Harmed individuals and accused individuals may [appeal](#appeals) the decision to the CoC Team up to thirty (30) days after a response was provided if they believe that the process was not followed properly. Appeals are not suspensive.
 
-## Report an issue in a space managed by the foundation
+To [appeal][] a decision, please email the [Code of Conduct Team (CoC Team)][CoC Team] at <coc-appeal@lists.openjsf.org>. TODO: create this address
```

Comment:
aside: it seems weird to me to have `lists.openjsf.org` rather than just `openjsf.org` on emails

Reply:
Agreed. @bensternthal, would that be possible?

Reply:
I created coc@openjsf.org which forwards to  report@lists.openjsf.org